### PR TITLE
user settings

### PR DESCRIPTION
**Pull request by xi**
This implements a basic user settings UI. You can currently edit username and password.

The edit page is not yet linked from the profile page as this requires #2588.

It might be a good idea to improve the UI, e.g. a second password field to avoid typos (like to one in the registration form).

### CODE REVIEW

**Comment by der-john** [comment]
we should change the wording, e.g. "settings" instead of "edit".

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.ts [hunk 1]
```diff
@@ -339,6 +339,10 @@ export class Service implements AdhTopLevelState.IAreaInput {
                 return;
             }
 
+            if (!self.has(resource.content_type, view, processType)) {
+                throw 404;
+            }
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.ts]: good idea.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.html [hunk 2]
```diff
@@ -11,11 +11,7 @@
     </adh-space>
 
     <adh-space data-key="user">
-        <div data-adh-moving-columns="" class="moving-columns">
-            <adh-moving-column>
-                <adh-user-detail-column></adh-user-detail-column>
-            </adh-moving-column>
-        </div>
+        <adh-user-workbench></adh-user-workbench>
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.html]: @xi: what was your motivation for calling it a workbench? The analogy with the processes' workbenches, right?

Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.html]: yes, exactly

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss [hunk 3]
```diff
@@ -128,3 +128,16 @@ We use a regular 12-column grid with fixed gutters.
...
+/*doc
+---
+title: Padding
+name: l-padding
+category: Layout
+---
+
+A generic layout class to add padding to the sides of an element.
+*/
+.l-padding {
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss]: why did you prefix it with "l-"?

Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss]: I wanted it to not be a regular component. The best category I could think of was layout.

**Comment by der-john** [comment]
looks good & works.

**Pull request merged**